tests/test_xmdf.cpp

```diff
@@ -206,7 +206,7 @@ TEST( MeshXmdfTest, CustomGroupsDataset )
   ASSERT_NE( g, nullptr );
 
   int meta_count = MDAL_G_metadataCount( g );
-  ASSERT_EQ( 1, meta_count );
+  ASSERT_EQ( 2, meta_count );
 
   const char *name = MDAL_G_name( g );
   EXPECT_EQ( std::string( "Vector Velocity" ), std::string( name ) );
```

Comment:
- can you please add test to check the metadata for timeunits and reference time for all ascii/binary/xmdf in case we have such testcase?

... so it means that we have no testcase for some other timeunits than defaults?, since you did not have to change any time ASSERT in any test. In that case I suggest to manually create a testcase (by let say copy-pasting some small ascii dat file) with other types of timeunit